kin-core/src/main/java/kin/core/PaymentsHistoryRequestParams.java

```diff
@@ -0,0 +1,129 @@
+package kin.core;
+
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Class which build optional params for the transaction payment history request.
```

Comment:
Represents optional params for payment history request, see <add link to the method>